bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock

```diff
@@ -6,15 +6,15 @@ PODS:
     - Flutter
   - package_info_plus (0.4.5):
     - Flutter
-  - "permission_handler (5.1.0+2)":
+  - permission_handler_apple (9.0.4):
```

Comment:
@luongvo `permission_handler` is already removed from the brick `template`. Can you help to update this file to get rid of `permission_handler` stuff? 🙏

Reply:
@manh-t good point, I cherry-picked it from a sample project with permission_handler enabled.

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/201/commits/5f5d1e25b12fa3b69b01bb632d55e2c6fe9604e2